syntax = "proto3";

package logging;

service LoggingService {
    rpc LogMessage (LogRequest) returns (LogResponse);
    rpc GetLogs (Empty) returns (LogList);
}

message LogRequest {
    string id = 1;
    string msg = 2;
}

message LogResponse {
    string status = 1;
}

message Empty {}

message LogList {
    repeated string logs = 1;
}
